docs/src/main/sphinx/connector/sql-update-limitation.fragment

```diff
@@ -0,0 +1,32 @@
+### UPDATE
+
+If a ``WHERE`` clause is specified, the ``UPDATE`` operation only works if the
+predicate in the clause can be fully pushed down to the data source.
```

Comment:
what does that mean "can be fully pushed down" ? How would you know as a user when looking at a statement or writing a statement?

Reply:
I think need to remove this statement